### PR TITLE
Added a new FallbackComponent

### PR DESCRIPTION
**Pull request by lorenzo**
It can be used to present payment methods that are not present in older versions of the SDK.

It will fallback to a webview.

### CODE REVIEW

File: KronorComponents/Fallback/SwiftUIView.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//
+//  FallbackWaitingView.swift
+//
+//
+//  Created by lorenzo on 2023-04-18.
+//
+
+import SwiftUI
+
+struct FallbackWaitingView: View {
```
Comment (stomas) [inline KronorComponents/Fallback/SwiftUIView.swift]: Filename should be the same as Struct is named.

File: KronorComponents/Common/EmbeddedPaymentViewModel.swift [hunk 2]
```diff
@@ -266,6 +307,11 @@ class EmbeddedPaymentViewModel: ObservableObject {
                             await self?.transition(.paymentRejected)
                         }
                     }
+                } else {
+                    print("noo")
```
Comment (stomas) [inline KronorComponents/Common/EmbeddedPaymentViewModel.swift]: Do we really need this?

**Review by stomas — APPROVED** [review]
Just minor issues, please fix and good to go.

File: KronorComponents/Common/EmbeddedPaymentViewModel.swift (hunk 2 above)
Comment (lorenzo) [reply KronorComponents/Common/EmbeddedPaymentViewModel.swift]: good catch

File: KronorComponents/Fallback/SwiftUIView.swift (hunk 1 above)
Comment (lorenzo) [reply KronorComponents/Fallback/SwiftUIView.swift]: oops, good catch!

**Pull request merged**